metrix-mapping/src/main/java/com/powsybl/metrix/mapping/NetworkPointWriter.java

```diff
@@ -99,6 +101,10 @@ private void mapToEquipmentVariable(Identifiable<?> identifiable, MappingVariabl
                     generator.setMinP((float) equipmentValue);
                 } else if (variable == EquipmentVariable.maxP) {
                     generator.setMaxP((float) equipmentValue);
+                } else if (variable == EquipmentVariable.voltageRegulatorOn) {
+                    generator.setVoltageRegulatorOn(equipmentValue == ON_VALUE);
+                } else if (variable == EquipmentVariable.targetV) {
+                    generator.setTargetV((float) equipmentValue);
```

Comment:
Why all the values are cast to float values? Is it legacy code, when IIDM used float for all numerical values?

Reply:
Ok. This remark is not related to the subject of the issue, we will do it later in a more general separate issue.